### PR TITLE
Switch art/design credits to use on-site username/ID instead of alias

### PR DESCRIPTION
**Pull request by itinerare**
- Add css to make images in parsed text not stretch the page, especially on mobile
- Switch artist and designer credits to using on-site usernames/IDs rather than aliases; user selection now a dropdown using selectize.
- Add a function akin to character ownership updating which, on verification or viewing of a user's profile, assigns any existing character image creators with the same alias as the user to the user's ID instead
- This function can also extract alias from dA URLs provided as credit to perform this operation
- Add a command to perform the same function for existing character image creators (`php artisan assign-art-credits-to-ids`) where possible (that is, where a user exists on the site with a given alias); aliases without associated accounts are converted to urls (assumes a dA account). The "alias" column can be safely dropped after this operation, if desired.
- Update masterlist/MYO masterlist search to use on-site usernames instead
- Add a helper function which formats user profile urls from various sites in the fashion of the old (username)@ dA links, and changed art/design credits to use it.
- Add a config file for sites including regex to facilitate the above/easy addition of sites

Tested locally - Requires migration

### CODE REVIEW

File: app/Http/Controllers/BrowseController.php [hunk 1]
```diff
@@ -283,16 +289,22 @@ public function getMyos(Request $request)
 
         // Searching on image properties
         if($request->get('species_id')) $imageQuery->where('species_id', $request->get('species_id'));
-        if($request->get('artists')) {
+        if($request->get('artist')) {
```
Comment (preimpression) [inline app/Http/Controllers/BrowseController.php:292]: Inconsistency in request for 'artist' vs 'artists'

File: app/Http/Controllers/BrowseController.php [hunk 2]
```diff
@@ -283,16 +289,22 @@ public function getMyos(Request $request)
...
-                $query->where('alias', $artistName);
+            // Usernames are prevented from containing spaces, but this is to deal with previously made accounts with spaces in names
+            $artistName = str_replace('%20', ' ', $artistName);
+            $artists = User::where('name', 'LIKE', '%' . $artistName . '%')->pluck('id')->toArray();
+            $imageQuery->whereHas('artists', function($query) use ($artists) {
+                $query->whereIn('user_id', $artists);
             });
         }
-        if($request->get('designers')) {
+        if($request->get('designer')) {
```
Comment (preimpression) [inline app/Http/Controllers/BrowseController.php:301]: Inconsistency in request for 'designer' vs 'designers'

**Review by preimpression — COMMENTED** [review]
Overall, looks good! Just the couple of weird inconsistencies we talked abt on Discord.

**Review by preimpression — APPROVED** [review]

**Review by Draginraptor — APPROVED** [review]
Sorry that it took a while, but I think the code looks good!

**Pull request merged**